test/linter/test-obsolete.js

```diff
@@ -0,0 +1,110 @@
+/* This file is a part of @mdn/browser-compat-data
+ * See LICENSE file for more information. */
+
+import chalk from 'chalk-template';
+import bcd from '../../index.js';
+const { browsers } = bcd;
+
+const exceptions = [
```

Comment:
Can you add a comment describing what `exceptions` means here?  I'm assuming it means "obsolete features that we want to keep", correct?

Reply:
I added the following comment:
```
/**
 * List of feature keys which are expected to fail this test.
 * This list is needed for two reasons:
 *  - some entries indeed should be removed (like api.VR*)
 *  - some entries are currently just stubs and should be expanded and will pass test later
 */
```

I plan to gradually fix these entries and remove exceptions. However, the `exceptions` variable itself is probably necessary since occasionally we have to add feature stubs for upcoming APIs.